ai/docs/GPT Online Model Training Documentation .md

```diff
@@ -0,0 +1,312 @@
+Model fine-tuning makes AI model following instructions better, as well as formatting output responses. This document describes how to use your own data to finetune an OpenAI GPT online model, just by taking 3 steps: data preparing, fine-tuning Job creating, and fine-tuned model calling. 
```

Comment:
Consider adding a brief overview or introduction at the beginning to provide context for readers who might be new to the topic.

Reply:
updated